docs/_sass/_main.scss

```diff
@@ -489,9 +490,10 @@ button {
     color: $color-gray-green;
     font-size: larger;
     display: inline;
-
-    &.fa-angle-right {
-        padding-left: 4px;
+    
+    &.icon-left, &.icon-right, &.icon-up, &.icon-down {
+        width: 16px;
```

Comment:
I think these should be 20px. I also think there is probably a cleaner way to do this? Let's create some kind of base icon class that uses 20px for dimensions and uses our standard icon fill color. This way we have one base class instead of needing to repeat these classes so many times.

Reply:
@shawnborton  for set color for icon. Actually, I set it by pass fill attr into path in svg file. If you want to use css to set color, I think we could covert our standard icon fill color to filter css. Do you have any solution to set it

Reply:
We should have a color variable for icons, if not it should be `#8B9C8F`

Reply:
Good point about how to set these... is there any way to do it via CSS?

Reply:
@shawnborton I saw that color variable for icons and I used this color to set color of svg icon. Other than filter css, I dont't know other way now.

Reply:
Okay yeah, let's just keep the icons colored as they are then. Thanks!